### PR TITLE
fix: remove DI schema constants from Marketo component

### PR DESCRIPTION
**Pull request by gonchar-ivan**
**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

remove DiSchemaConstants class usage from Marketo

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/805/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/805/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/806/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/806/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/807/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/807/)

File: components/components-netsuite/components-netsuite-runtime/src/main/java/org/talend/components/netsuite/NetSuiteDatasetRuntimeImpl.java [hunk 1]
```diff
@@ -386,21 +385,21 @@ public static Schema inferSchemaForType(String name, List<FieldDesc> fieldDescLi
...
+                avroField.addProp(SchemaConstants.TALEND_COLUMN_DB_TYPE, customFieldRefType.getTypeName());
 
                 if (customFieldRefType == CustomFieldRefType.DATE) {
                     avroField.addProp(SchemaConstants.TALEND_COLUMN_PATTERN, "yyyy-MM-dd'T'HH:mm:ss'.000Z'");
                 }
 
                 NsRef ref = customFieldInfo.getCustomizationRef();
                 if (StringUtils.isNotEmpty(ref.getName())) {
-                    avroField.addProp(DiSchemaConstants.TALEND6_COMMENT, ref.getName());
+                    avroField.addProp(NetSuiteSchemaConstants.TALEND6_COMMENT, ref.getName());
```
Comment (wwang-talend) [inline components/components-netsuite/components-netsuite-runtime/src/main/java/org/talend/components/netsuite/NetSuiteDatasetRuntimeImpl.java]: why it is in NetSuiteSchemaConstants? not common?

**Pull request merged**